app/utils/ajax-helpers.ts

```diff
@@ -1,5 +1,6 @@
 import config from 'ember-get-config';
 import $ from 'jquery';
+import { Promise as EmberPromise } from 'rsvp';
```

Comment:
Maybe just `import RSVP from 'rsvp''` here and `return new RSVP.Promise(...` below?

Reply:
Yeah, that's cleaner 👍